modin/core/dataframe/pandas/dataframe/dataframe.py

```diff
@@ -990,6 +990,19 @@ def _take_2d_positional(
         if col_positions is None and row_positions is None:
             return self.copy()
 
+        if col_positions is None and row_positions is not None:
+            # fast path
```

Comment:
an explanation of why it's a fast path and what's going on would be useful IMHO

Reply:
@vnlitvinov comment added, synthetic benchmark result at the top.